sockeye/model.py

```diff
@@ -42,6 +42,8 @@ class ModelConfig(Config):
     :param max_seq_len_target: Maximum target sequence length to unroll during training.
     :param vocab_source_size: Source vocabulary size.
     :param vocab_target_size: Target vocabulary size.
+    :param num_factors: The number of source factors.
```

Comment:
do we need this in the top-level config? Can't we access it from the config_embed_source as well? This might make it clearer that source factors are only relevant for the source embedding.

Reply:
sure